src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int i = 1; i <= 3; i++) {
+            Lottery ball = new Lottery();
```

Comment:
Let's create Lottery instance outside the loop and call it lottery instead of ball